### PR TITLE
fixed page redirect after deleting and editing book

### PR DESCRIPTION
**Pull request by zaidoon1**
Previously the page wouldn't redirect back to index page properly after editing and deleting a book. This has now been fixed.

Additionally cleans up .gitignore file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ target/
 .settings/org.eclipse.core.resources.prefs
 .settings/org.eclipse.jdt.core.prefs
 .vscode/launch.json
+.vscode/settings.json
```
Comment (abungay) [inline .gitignore]: should we just ignore the whole .vscode/ folder and the whole .settings folder?

Comment (zaidoon1) [reply .gitignore]: i think that is better

Comment (zaidoon1) [reply .gitignore]: the .vscode folder is now being ignored

**Review by abungay — APPROVED** [review]

**Review by Hajar1 — APPROVED** [review]
looks like needed changes

**Pull request merged**